Formula/bazarr.rb

```diff
@@ -99,32 +99,8 @@ def post_install
   end
 
   test do
-    require "open3"
-    require "timeout"
-
-    system "#{bin}/bazarr", "--help"
-
-    config_file = testpath/"config/config.ini"
-    config_file.write <<~EOS
-      [backup]
-      folder = #{testpath}/custom_backup
-    EOS
-
-    port = free_port
-
-    Open3.popen3("#{bin}/bazarr", "--config", testpath, "-p", port.to_s) do |_, _, stderr, wait_thr|
```

Comment:
I mean, perhaps we can add a `--no-update` flag here, instead of removing the whole server test? In that case it won't throw an error, IIUC.

- https://github.com/morpheus65535/bazarr/blob/v1.2.4/bazarr/main.py#L28-L35
- https://github.com/morpheus65535/bazarr/blob/v1.2.4/bazarr/app/check_update.py

Reply:
👍